### PR TITLE
[cli] Minor `vc inspect` visual updates

### PR DESCRIPTION
**Pull request by MatthewStanciu**
Featuring:

- Aliases are now shown
- `readyState` –> `status` in "General" section
- `createdAt` –> `created` in "General" section
- Update `stateString` to support more states & be more visually appealing

### Before

Screen Shot 2022-07-14 at 3 10 40 PM

### After

Screen Shot 2022-07-14 at 3 10 29 PM

#### Tests

- [x] The code changed/added as part of this PR has been covered with tests
- [x] All tests pass locally with `yarn test-unit`

#### Code Review

- [ ] This PR has a concise title and thorough description useful to a reviewer
- [ ] Issue from task tracker has a link to this PR

### CODE REVIEW

**Review by EndangeredMassa — APPROVED** [review]

File: packages/cli/src/commands/inspect.ts [hunk 1]
```diff
@@ -165,19 +184,24 @@ export default async function main(client: Client) {
...
-
+      return chalk.red(CIRCLE) + sTitle;
     case 'READY':
-      return s;
-
+      return chalk.green(CIRCLE) + sTitle;
+    case 'QUEUED':
+      return chalk.white(CIRCLE) + sTitle;
+    case 'CANCELED':
+      return chalk.gray(sTitle);
```
Comment (styfle) [inline packages/cli/src/commands/inspect.ts]:
```suggestion
      return chalk.gray(CIRCLE) + sTitle;
```

I think this should say canceled, right?

File: packages/cli/src/commands/inspect.ts [hunk 2]
```diff
@@ -165,19 +184,24 @@ export default async function main(client: Client) {
...
     default:
-      return chalk.gray(s || 'UNKNOWN');
+      return chalk.gray('  --');
```
Comment (styfle) [inline packages/cli/src/commands/inspect.ts]: Lets keep this `UNKNOWN`

File: packages/cli/src/commands/inspect.ts [hunk 3]
```diff
@@ -165,19 +184,24 @@ export default async function main(client: Client) {
...
     case 'ERROR':
-      return chalk.red(s);
-
+      return chalk.red(CIRCLE) + sTitle;
     case 'READY':
-      return s;
-
+      return chalk.green(CIRCLE) + sTitle;
+    case 'QUEUED':
+      return chalk.white(CIRCLE) + sTitle;
```
Comment (styfle) [inline packages/cli/src/commands/inspect.ts]: I believe we use gray for queued status on the dashboard so best to match that

**Comment by TooTallNate** [comment]
How about using `https://` for the deployment URL and aliases?

**Comment by TooTallNate** [comment]
What do you think about dropping the "Ready" in the Builds section? Seems redundant to me.

**Pull request merged**